docs/protocol/on-chain/channel-lifecycle.mdx

```diff
@@ -0,0 +1,375 @@
+---
+sidebar_position: 3
+title: "Channel Lifecycle"
+---
+
+import Tooltip from '@site/src/components/Tooltip';
+
+# Channel Lifecycle
+
+## Creation Phase
+
+**Purpose**: Initiate a new <Tooltip content="A secure communication pathway between participants that locks funds in an on-chain smart contract while enabling off-chain state updates.">channel</Tooltip> with specified <Tooltip content="An entity (identified by a wallet address) that is part of a channel.">participants</Tooltip> and initial funding.
+
+**Process**:
+
+1. The <Tooltip content="Most often a light client willing to fund a ledger account with the Broker.">Creator</Tooltip> MUST:
+   - Construct a Channel configuration with <Tooltip content="An entity (identified by a wallet address) that is part of a channel.">participants</Tooltip>, <Tooltip content="A smart contract that validates state transitions according to application-specific rules.">adjudicator</Tooltip>, challenge period, and nonce
+   - Prepare an initial <Tooltip content="A snapshot of the channel at a point in time, including fund allocations and application-specific data.">State</Tooltip> where `state.data` contains the magic number `CHANOPEN` (7877)
+   - Define expected token deposits for all participants in `state.allocations`
+   - Compute the funding <Tooltip content="A cryptographic hash of a channel state, used for signature verification.">stateHash</Tooltip> of this initial state
+   - Include Creator's signature in `state.sigs` at position 0
+   - Call the `create` function with the channel configuration and initial signed state
+
```

Comment:
I think it is better to specify an optional "implicit join" mechanism right here as a "callout", and not below in the "Joining phase" section. This is for when a Creator obtains the second participant signature over the initial state, and supplies it alongside their one. If the second participant has enough funds to be deposited into the channel, it will transition to "Active" status.

Reply:
[f889dd9d34e1744c4d0d2598c3c590f87acb676b](https://github.com/layer-3/docs/pull/69/commits/f889dd9d34e1744c4d0d2598c3c590f87acb676b)